src/server/scripts/Northrend/Naxxramas/boss_sapphiron.cpp

```diff
@@ -385,6 +422,41 @@ class boss_sapphiron : public CreatureScript
         }
 };
 
+struct npc_sapphiron_blizzard : public ScriptedAI
+{
+    npc_sapphiron_blizzard(Creature* creature) : ScriptedAI(creature) { }
+
+    void Reset() override
+    {
+        me->SetReactState(REACT_PASSIVE);
+        _scheduler.Schedule(Seconds(3), [this](TaskContext chill)
```

Comment:
You just missed a chance to use the new chrono literals D:

Reply:
idk how new chrono works, example? xD

Reply:
```cpp
_scheduler.Schedule(3s, std::function<void(TaskContext)>(...));
```